Atomic/atomic.py

```diff
@@ -272,6 +360,11 @@ def set_args(self, args):
             self.spc = False
 
         try:
+            self.system = args.system
+        except:
+            self.system = False
```

Comment:
I looked into this, but since I use `self.system` short after, I preferred to keep this as it makes the code cleaner instead of checking if `self.args` has `system` every time.  It is similar to .spc defined just above